### PR TITLE
Sticky timer + change button size

### PR DESCRIPTION
**Pull request by tjw0612**
## Why

This pull request is needed because:

Fix button shape/size
image

Added sticky timer
image

### CODE REVIEW

File: app/controllers/chomp_sessions_controller.rb [hunk 1]
```diff
@@ -15,7 +15,7 @@ def new
   end
 
   def create
-    @chomp_session = ChompSession.new(chomp_params)
+  @chomp_session = ChompSession.new(chomp_params)
```
Comment (mxng) [inline app/controllers/chomp_sessions_controller.rb]: indentation

**Review by mxng — APPROVED** [review]
looks ok

File: app/views/chomp_sessions/_timer.html.erb [hunk 2]
```diff
@@ -1,6 +1,36 @@
 <div class="timer">
-  <p class="time-desc time-desc-title">
-  <span><%= ((@chomp_session.updated_at + @chomp_session.session_expiry.hour - Time.now)/3600).floor %>H
-  <%= ((@chomp_session.updated_at + @chomp_session.session_expiry.hour - Time.now)% 3600 /60).floor %>Min</span> before poll closes</p>
-  <div id="clock"></div>
+  <div class="time-desc time-desc-title w-100">
```
Comment (amelialimyh) [inline app/views/chomp_sessions/_timer.html.erb:2]: Might want to add some space in-between the timer and the text

**Review by amelialimyh — COMMENTED** [review]
Except for the spacing issue, the rest is fine.

File: app/views/chomp_sessions/_timer.html.erb (hunk 2 above)
Comment (mxng) [reply app/views/chomp_sessions/_timer.html.erb:2]: i will add it in another branch ba

**Review by mxng — CHANGES_REQUESTED** [review]
got conflictttt

**Review by mxng — APPROVED** [review]

**Pull request merged**